### PR TITLE
[Spree Upgrade] Fix fees tests in models/spree/order_spec

### PR DESCRIPTION
**Pull request by luisramos0**
#### What? Why?

Continues #2685 

#### What should we test?
Fixes 3 more tests in order_spec.

### CODE REVIEW

**Comment by luisramos0** [comment]
This PR is ready for review now.

**Comment by luisramos0** [comment]
I have resolved conflicts and removed rubocop upgrade from this PR.

File: spec/models/spree/order_spec.rb [hunk 1]
```diff
@@ -676,6 +677,7 @@
         # Setting quantity of an item to zero
         order.update_attributes(line_items_attributes: [{id: order.line_items.first.id, quantity: 0}])
 
+
```
Comment (sauloperez) [inline spec/models/spree/order_spec.rb]: 🔥

File: app/models/spree/payment_decorator.rb [hunk 2]
```diff
@@ -32,11 +32,7 @@ def adjustment_label
 
     # This is called by the calculator of a payment method
     def line_items
-      if order.complete? && Spree::Config[:track_inventory_levels]
-        order.line_items.select { |li| inventory_units.pluck(:variant_id).include?(li.variant_id) }
```
Comment (sauloperez) [inline app/models/spree/payment_decorator.rb]: why is this not needed anymore?

File: spec/models/spree/order_spec.rb (hunk 1 above)
Comment (luisramos0) [reply spec/models/spree/order_spec.rb]: :-)

File: app/models/spree/payment_decorator.rb (hunk 2 above)
Comment (luisramos0) [reply app/models/spree/payment_decorator.rb]: this is in the scope of adjustments. see calculator_decorator.line_items_for..
shipment like payment has one adjustment.
this was incorrectly copied from shipment where there are inventory_units to be tracked.
payment doesnt have inventory_units ;-)

**Comment by luisramos0** [comment]
I rebased and removed extra empty line.

**Comment by mkllnk** [comment]
@luisramos0 There is a conflict to resolve now. Please rebase again. :-)

**Comment by luisramos0** [comment]
thanks for the heads up Maikel. Rebased, conflict resolved.

File: app/models/spree/payment_decorator.rb (hunk 2 above)
Comment (sauloperez) [reply app/models/spree/payment_decorator.rb]: :scream: Your're becoming our CAO, chief adjustments officer :trollface:

Comment (luisramos0) [reply app/models/spree/payment_decorator.rb]: ahaha, CAO means dog in Portuguese 😆

**Pull request merged**